File: wrappers/java/src/main/java/org/hyperledger/indy/sdk/WalletValueNotFoundException.java
package org.hyperledger.indy.sdk;

public class WalletValueNotFoundException extends IndyException
{
	private final static String message = "No value with the specified key exists in the wallet from which it was requested.";

	WalletValueNotFoundException(int sdkErrorCode) 
    {
    	super(message, sdkErrorCode);
    }
}

File: wrappers/dotnet/indy-sdk-dotnet/WalletApi/WalletValueNotFoundException.cs
namespace Hyperledger.Indy.WalletApi
{
    /// <summary>
    /// Exception thrown when requesting a value from a wallet and the specified key does not exist.
    /// </summary>
    public class WalletValueNotFoundException : IndyException
    {
        const string message = "No value with the specified key exists in the wallet from which it was requested.";

        /// <summary>
        /// Initializes a new WalletValueNotFoundException.
        /// </summary>
        internal WalletValueNotFoundException() : base(message, (int)ErrorCode.WalletNotFoundError)
        {

        }
    }

}
